src/github.rs

```diff
@@ -136,14 +136,49 @@ impl GitHubApi {
             }
         ";
 
+        let cant_resolve = |e: &Error| e.to_string().contains("Could not resolve to a node");
+
         let mut result = HashMap::new();
         for chunk in ids.chunks(100) {
-            let res: GraphNodes<Usernames> = self.graphql(
+            let res: GraphNodes<Usernames> = match self.graphql(
                 QUERY,
                 Params {
                     ids: chunk.iter().map(|id| user_node_id(*id)).collect(),
                 },
-            )?;
+            ) {
+                Ok(res) => res,
+                Err(e) => {
+                    if cant_resolve(&e) {
+                        // This error happens when a user is deleted. Provide
+                        // a more helpful error message to pinpoint it:
+                        for id in chunk {
```

Comment:
Is doing a binary search worth the potential added complexity? Doing potentially 100 requests for the error message sounds like quite a lot...

Reply:
Yes exactly. Binary search would definitely help reduce the number of requests. I agree we're probably ok just making the possible ~100 requests, but we *might* run into issues in the future. CI runs every hour, and I've also considered making it every 10 minutes (to make sure things are always in sync).

Reply:
Ah, I didn't know about the scheduled jobs.

Now that I think about it, I'm not actually sure what the limit is here. I think the rust-lang org is an Enterprise account, which I think has a limit of 15,000 requests per hour per repo using the GitHub Actions auto-generated token.